client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -1,23 +1,224 @@
 /**
  * External dependencies
  */
-import { localize } from 'i18n-calypso';
+import { bindActionCreators } from 'redux';
+import { translate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
+import { fetchEmailSettings } from 'woocommerce/state/sites/settings/email/actions';
+import {
+	getEmailSettings,
+	areEmailSettingsLoading,
+	areEmailSettingsLoaded,
+} from 'woocommerce/state/sites/settings/email/selectors';
+import Card from 'components/card';
+import CustomerNotification from './components/customer-notification';
+import InternalNotification from './components/internal-notification';
+import NotificationsOrigin from './components/notifications-origin';
+
+const fromName =
+	{
+		field: 'woocommerce_email_from_address',
+		title: translate( 'From name' ),
+		subtitle: translate( 'Emails will appear in recipients inpboxses \'from\' this name.' ),
+	};
+const fromAddress =
+	{
+		field: 'woocommerce_email_from_name',
+		title: translate( 'From address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **87** times:
`translate( 'Address' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).